roles/common/tasks/debian.yml

```diff
@@ -196,3 +199,11 @@
       - unzip
       - curl
   tags: package
+
+- name: Get Java Version
+  shell: java -version
+  register: version_output
+
+- name: Print Java Version
+  debug:
+    msg: "Current Java Version is: {{version_output.stderr_lines[0]}}"
```

Comment:
Does it not output in stdout rather than stderr?
Lets halt the execution if there is no Java configured at this stage.

Reply:
It outputs in stderr, though if I'll use Java --version instead of Java -version, it would output in stdout. But that was not working for Java 8 scenarios. Based on the flow of the tasks, there would always be Java configured at this stage since we're already halting in custom_java_install.yml or installing otherwise. Should I also add a halt over here?